README.md

```diff
@@ -69,15 +69,23 @@ Pass other parameters to jest after the `--` flag. For example,
 lwc-jest --debug -- --no-cache
 ```
 
+### Debugging in VSCode
+
+If you prefer to debug directly inside the VSCode IDE, follow these steps:
+- From the VSCode dropdowns, select Debug --> Add Configuration...
+- Replace the contents of generated `launch.json` with the [`Debug Jest Tests` configuration](https://jestjs.io/docs/en/troubleshooting#debugging-in-vs-code) provided by Jest
+- Add a `jest.config.json` file to the root of the SFDX project as described [here](#overriding-jest-config)
```

Comment:
We need a `jest.config.js` here, not a json file. also replace `SFDX` with `Salesforce DX`.